### PR TITLE
Implements sharify and inquiries page styling

### PR DESCRIPTION
**Pull request by l2succes**

### CODE REVIEW

File: src/apps/loyalty/containers/inquiries/index.tsx [hunk 1]
```diff
@@ -81,14 +101,20 @@ class Inquiries extends React.Component<RelayProps, State> {
     return (
       <Container>
         <Nav>
-          <NavItem href="https://www.artsy.net">Back To Artsy</NavItem>
+          <NavItem href="/">Back To Artsy</NavItem>
         </Nav>
-        <Title>Please select all works your purchased</Title>
+        <Header>
+          <Title titleSize="large" className="header-title">Please select all works your purchased</Title>
```
Comment (alloy) [inline src/apps/loyalty/containers/inquiries/index.tsx]: Opportunity to fix #58.

File: src/apps/loyalty/server/index.tsx [hunk 2]
```diff
@@ -20,15 +20,19 @@ import { RelayMiddleware } from "./relay"
 
 const app = express()
 const artsyPassport = require("artsy-passport")
+const sharify = require("sharify")
```
Comment (alloy) [inline src/apps/loyalty/server/index.tsx]: Are these not loadable with `import`?

File: src/apps/loyalty/containers/inquiries/index.tsx [hunk 3]
```diff
@@ -2,27 +2,45 @@ import * as React from "react"
...
-import TextArea from "../../../../components/text_area"
-import Title from "../../../../components/title"
+import Button from "components/buttons/inverted"
+import { Col, Row } from "components/grid"
+import Artwork from "components/inquiry_artwork"
+import Nav from "components/nav"
+import NavItem from "components/nav_item"
+import Text from "components/text"
+import TextArea from "components/text_area"
+import Title from "components/title"
```
Comment (alloy) [inline src/apps/loyalty/containers/inquiries/index.tsx]: Sweeeeet

File: src/apps/loyalty/server/index.tsx (hunk 2 above)
Comment (l2succes) [inline src/apps/loyalty/server/index.tsx]: We would need to create typings for those, which is doable but not super necessary for now

Comment (alloy) [inline src/apps/loyalty/server/index.tsx]: You can use ES6 imports without typings being available.

**Pull request merged**